tests/test_core.py

```diff
@@ -186,65 +195,74 @@ def __init__(self, data_home=None, version="default"):
                     "1": core.Index(
                         "blah_1.json", url="https://google.com", checksum="asdf"
                     ),
-                    "2": core.Index("blah_2.json"),
-                    "real": core.Index("beatles_index_1.2.json"),
+                    "real": core.Index("acousticbrainz_genre_index_sample.json"),
```

Comment:
We are changing the name of this index in https://github.com/mir-dataset-loaders/mirdata/pull/641/files#diff-69bfb079ff64b5be1e0fc8190bd65853a62bb32725951512bcf2536509362e40 to `acousticbrainz_genre_index_1.0_sample.json` decide what to do, if use another dataset as example, or we change it in the other PR (just a heads up so we don't forget) CC @genisplaja